src/components/TextInput/TextInput.vue

```diff
@@ -168,12 +172,19 @@ export default {
     copyToClipboard () {
       this.$refs.input.select();
       document.execCommand('copy');
+      this.$emit('copy');
```

Comment:
Just wondering about use cases, should we emit the copied text too?

Reply:
@juanfriss In Settings -> API Keys screen we show a confirmation message that key was copied for 3 seconds.
I'm not sure if I should put screenshots here because its a public lib.